parachain/pallets/inbound-queue/src/test.rs

```diff
@@ -2,325 +2,36 @@
-			hex!("00000000000000000000000000000000000000000000000000000000000003e8").into(),
-			// message id
-			hex!("afad3c9777134532ae230b4fad334eef2e0dacbb965920412a7eaa59b07d640f").into(),
-		],
-		// Nonce + Payload
-		data: hex!("00000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000001e000f000000000000000087d1f7fdfee7f651fabc8bfcb6e086c278b77a7d0000").into(),
-	}
-}
-
-fn mock_event_log_invalid_channel() -> Log {
-	Log {
-		// gateway address
-		address: hex!("eda338e4dc46038493b885327842fd3e301cab39").into(),
-		topics: vec![
-			hex!("5066fbba677e15936860e04088ca4cad3acd4c19706962196a5346f1457f7169").into(),
-			// channel id
-			hex!("00000000000000000000000000000000000000000000000000000000000003e9").into(),
-			// message id
-			hex!("afad3c9777134532ae230b4fad334eef2e0dacbb965920412a7eaa59b07d640f").into(),
-		],
-		// Nonce + Payload
-		data: hex!("00000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000001e000f000000000000000087d1f7fdfee7f651fabc8bfcb6e086c278b77a7d0000").into(),
-	}
-}
-
-fn mock_event_log_invalid_gateway() -> Log {
-	Log {
-		// gateway address
-		address: H160::zero(),
-		topics: vec![
-			hex!("5066fbba677e15936860e04088ca4cad3acd4c19706962196a5346f1457f7169").into(),
-			// channel id
-			hex!("00000000000000000000000000000000000000000000000000000000000003e8").into(),
-			// message id
-			hex!("afad3c9777134532ae230b4fad334eef2e0dacbb965920412a7eaa59b07d640f").into(),
-		],
-		// Nonce + Payload
-		data: hex!("00000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000001e000f000000000000000087d1f7fdfee7f651fabc8bfcb6e086c278b77a7d0000").into(),
-	}
-}
-
-const ASSET_HUB_PARAID: u32 = 1000u32;
-const TEMPLATE_PARAID: u32 = 1001u32;
+use crate::mock::{
+	expect_events, mock_event_log, mock_event_log_invalid_channel, mock_event_log_invalid_gateway,
+	new_tester, AccountId, Balances, InboundQueue, RuntimeOrigin, Test, ASSET_HUB_PARAID,
+};
 
 #[test]
 fn test_submit_happy_path() {
 	new_tester().execute_with(|| {
 		let relayer: AccountId = Keyring::Bob.into();
 		let origin = RuntimeOrigin::signed(relayer);
 
-		// Deposit funds into sovereign account of Asset Hub (Statemint)
-		let sovereign_account = sibling_sovereign_account::<Test>(ASSET_HUB_PARAID.into());
-		println!("account: {}", sovereign_account);
-		let _ = Balances::mint_into(&sovereign_account, 10000);
-
 		// Submit message
-		let message = Message {
-			event_log: mock_event_log(),
-			proof: Proof {
-				block_hash: Default::default(),
-				tx_index: Default::default(),
-				data: Default::default(),
-			},
-		};
+		let message = make_create_message().message;
```

Comment:
I feel like this new `make_create_message` fixture makes our tests inconsistent.

Previously, all tests were using `mock_event_log_*` fixtures, and now this test is using a different fixture, which returns different data types.